scripts/migrate-grid.mjs

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env node
+
+/* eslint-disable no-console */
+
+import fs from 'fs'
+import chalk from 'chalk'
+
+const args = process.argv.slice(2)
+const dryRun = args.includes('--dry-run')
+const files = args.filter((arg) => !arg.startsWith('--'))
+
+if (files.length === 0) {
+  console.error('❌ No input files provided.')
+  process.exit(1)
+}
+
+// --- Mapping Rules ---
+
+const exactReplacements = {
+  'vads-l-grid-container': 'vads-grid-container',
+  'vads-l-row': 'vads-grid-row',
+  'usa-grid-full': 'vads-grid-container', // Needs to come before usa-grid because replacing usa-grid first will mess this one up
+  'usa-grid': 'vads-grid-container',
+  'usa-width-one-half': 'vads-grid-col-6',
+  'usa-width-one-third': 'vads-grid-col-4',
+  'usa-width-two-thirds': 'vads-grid-col-8',
+  // Add more as needed
+}
+
+const prefixReplacements = {
+  'vads-l-col--': 'vads-grid-col-',
+}
+
+const breakpointReplacements = {
+  'xsmall-screen:vads-l-': 'mobile:vads-grid-',
+  'small-screen:vads-l-': 'mobile-lg:vads-grid-',
+  'medium-screen:vads-l-': 'tablet:vads-grid-',
+  'small-desktop-screen:vads-l-': 'desktop:vads-grid-',
+  'large-screen:vads-l-': 'desktop-lg:vads-grid-',
+}
+
+const foundationRegexes = [
+  /\b(row|column|columns)\b/g,
+  /\bsmall-[\w-]+\b/g,
+  /\bmedium-[\w-]+\b/g,
```

Comment:
Only thought here is whether or not it'd be better to just look for numeric values instead of alphanumeric. Quick search in vets-website shows we might have several instances of other random classes that are prefixed with `medium-` but end with things other than numbers. Would using this pattern pick those things up?